### ISSUE TITLE
stomp sample not running

### ISSUE THREAD

**Issue by aknuth** [Atmosphere/atmosphere-samples#42]
Hi,

tried to run the stomp pubsub sample on the stomp/pubsub dir using mvn jetty:run

always get the following error:
```
217 [main] ERROR org.atmosphere.cpr.AtmosphereFramework - Failed to initialize Atmosphere Framework
java.lang.NoSuchMethodError: org.atmosphere.cpr.AtmosphereResourceSessionFactory.getDefault()Lorg/atmosphere/cpr/AtmosphereResourceSessionFactory;
	at org.atmosphere.stomp.interceptor.FrameInterceptor.configure(FrameInterceptor.java:183)
        .....
```
any idea on that ? (codebase is master)

using nettosphere yields the same result :-(

thanks a lot

**Comment by jfarcand** [comment]
@gdrouet Do you hav ethe cycle to upate and cut a release with the latest Atmosphere's API?

**Comment by aknuth** [comment]
looks that atmosphere-stomp 0.3-SNAPSHOT does not have this problem with the actual atmosphere Snapshot.
But there is another error which might depend on my ServletEngine configuration. The Stomp HandlerHelper cannot find the atmosphere Handler for the destination-1 etc - and this might be the result of AnnotationScanning process.
Does the AnnotationsScanner anyway scans for StompEndpoint ?

**Comment by jfarcand** [comment]
Fixed with 0.3 version
